examples/pytorch/IterativePruning/MLproject

```diff
@@ -0,0 +1,17 @@
+name: alexnet-cifar10-classification
+
+conda_env: conda.yaml
+
+entry_points:
+  main:
+    parameters:
+      max_epochs: {type: int, default: 3}
+      mlflow_run_name: {type: str, default: 'BaseModel'}
+      mlflow_experiment_name: {type: str, default: "Default"}
+
+
+    command: |
+          python alexnet.py \
+            --max_epochs {max_epochs} \
+            --mlflow_run_name {mlflow_run_name} \
+            --mlflow_experiment_name {mlflow_experiment_name} \
```

Comment:
@ankan94 Please remove the last backslash in the command.

Reply:
@HamidShojanazeri This seems to be a No module error. Looks like Prettytable needs installation. I have run the script  and is working fine for me. I am updating the conda.yaml file to have prettytable requirement. Please let me know if it works for you then. Thanks.